ganttproject/src/main/java/biz/ganttproject/storage/cloud/GPCloudHttpImpl.kt

```diff
@@ -262,18 +267,27 @@ class WebSocketClient {
   private val structureChangeListeners = mutableListOf<(Any) -> Unit>()
   private val lockStatusChangeListeners = mutableListOf<(ObjectNode) -> Unit>()
   private val contentChangeListeners = mutableListOf<(ObjectNode) -> Unit>()
+  private val xlogReceivedListeners = mutableListOf<(ObjectNode) -> Unit>()
   private var listeningDocument: GPCloudDocument? = null
 
-  fun start() {
+  private fun getWebSocketUrl(): String {
+    return if (isColloboqueLocalTest()) {
```

Comment:
more Kotlin'ish style: 

```suggestion
  private fun getWebSocketUrl() = if (isColloboqueLocalTest()) {
```

Reply:
Thanks!